tests/compatibility-tests/src/main/resources/exportimport/import1X.groovy

```diff
@@ -0,0 +1,11 @@
+import org.apache.activemq.artemis.cli.commands.ActionContext
```

Comment:
License header missing, should this not be picked up by RAT report? Do we need to make rat look at groovy files?

Reply:
It was picked actually.

Reply:
the reason is failed...

Reply:
@dudaerich the reason we have export / export1X is because the package names are different.

I could have fixed it differently with a different script just to instantiate the export.. but the script was so small that I decided just to have different versions for each branch.


the script wouldn't load (or compile) if I had any sort of if inside.